libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -470,12 +482,16 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
     this.resultsListId = `sky-autocomplete-list-${id}`;
     this.resultsWrapperId = `sky-autocomplete-wrapper-${id}`;
 
+    this.#activeElementIndex = -1;
```

Comment:
I don't think you need to change it in this PR, but for future reference I think it's OK to assign static values when declaring private fields. It's only when you assign one private field's value to another private field's value during declaration that creates some ambiguity as to which is set first.